### PR TITLE
[Investigation] authentication + authroization service

### PR DESCRIPTION
**Pull request by yangxvlin**
as described in https://github.com/ml-for-good/automl/pull/31

### CODE REVIEW

**Comment by free-y** [comment]
I can't see the image files in this PR, is it expected?
Below is an example image link:
https://raw.githubusercontent.com/ml-for-good/automl/dcd17b45145342cdea66dcedacbb2bfc543adb65/Investigation/Java/Java-auth/auth_server/imgs/1.png

**Comment by yangxvlin** [comment]
I think this is because 
```
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/1.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/10.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/11.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/12.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/2.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/3.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/4.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/5.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/6.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/7.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/8.png.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in Investigation/Java/Java-auth/auth_server/img/9.png.
The file will have its original line endings in your working directory
```
when I commit my code. The image works fine on my machine.

**Comment by yangxvlin** [comment]
will try to change png to jpg

**Comment by free-y** [comment]
Please try to fix the display issue of images, there is another case that png can be show correctly. Please check it: https://github.com/ml-for-good/automl/pull/76/files

**Comment by yangxvlin** [comment]
fixed by converting png to jpg

File: Investigation/Java/Java-auth/auth_server/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    - ```json
+      {
+          ...,
+          "token_endpoint": "http://localhost:8080/realms/automl_dev/protocol/  openid-connect/token",
+          ...
+      }
+      ```
+4. try: `POST http://localhost:8080/realms/automl_dev/protocol/openid-connect/token`
+    - with:  
+        <img src="./img/2.jpg" width="60%"/>
```
Comment (free-y) [inline Investigation/Java/Java-auth/auth_server/README.md:23]: img 2 can't display

Comment (yangxvlin) [reply Investigation/Java/Java-auth/auth_server/README.md]: resolved in [1a59426](https://github.com/ml-for-good/automl/pull/52/commits/1a594260fb18cbeba342dbd1a2faeccfc9651de7)

**Pull request merged**